az-core/src/main/java/azkaban/Constants.java

```diff
@@ -311,4 +311,13 @@
     public static final String START_TIME = "startTime";
     public static final String TRIGGER_INSTANCE_ID = "triggerInstanceId";
   }
+
+
+  public static class DefaultValue {
+
+    public static final int PROJECT_DIR_CLEANUP_START_THRESHOLD = 90;
+    public static final int PROJECT_DIR_CLEANUP_STOP_THRESHOLD = 60;
+    //128G
+    public static final long PROJECT_DIR_MAX_SIZE = 12800;
```

Comment:
Is it in MB? 
Should it be 128_000 then?

Reply:
thanks.